waterlog/src/Stylesheets/_alerts.scss

```diff
@@ -6,10 +13,15 @@ table {
   margin-bottom: 10px;
 }
 
+td{
+  padding: 2% 2% 2% 2%;
```

Comment:
You can use 2% only once and it will apply to the four corners of the td element